### ISSUE TITLE
Close button does not close

### ISSUE THREAD

**Issue by basz** [sethwebster/ember-cli-new-version#106]
ignoredVersions is an array

pushing an ignored version to it does not trigger a recalculation of the message.

i suggest to change it to 

```
@tracked ignoredVersions = [];
```

and use 

```
  ignoreVersion(version) {
    this.ignoredVersions = [...this.ignoredVersions, version];
  }
```

to add a version to be ignored.

**Comment by basz** [comment]
also change

```
- this.ignoredVersions.lastObject || this.currentVersion
+ this.ignoredVersions[this.ignoredVersions.length - 1] || this.currentVersion
```

**Comment by knownasilya** [comment]
@basz do you have time to submit a pr? Probably can release a fix faster if you do.

**Comment by basz** [comment]
I'm not using the close button myself. But maybe after weekend.

**Comment by knownasilya** [comment]
Ok, will probably get to it before then myself

**Comment by knownasilya** [comment]
Released in v4.0.1
